### PR TITLE
Fix citation section

### PR DESCRIPTION
**Pull request by orzechow**
- **Fix broken github icon paths**
- **Fix rendering of bibtex code blocks**

### CODE REVIEW

**Comment by orzechow** [comment]
@ll-nick

File: README.md [hunk 1]
```diff
@@ -303,6 +303,7 @@ Nick Le Large&nbsp;
 
 If you use arbitration graphs in your research, we would be pleased if you cite our work:
 
+{% raw %}
```
Comment (ll-nick) [inline README.md:306]: I assume this is an adjustment for the pages? On the GitHub rendered readme this doesn't do anything and I'd assume that people will often see this one first before going to the website...

Comment (orzechow) [reply README.md:306]: Good catch, that's a no-go!

I assume'd the GitHub Readme rendering uses Jekyll as well. Probably it has a more restrictive setup. 

The issue on Pages is that our bibtex titles with double curly brackets are eaten by Liquid (Jekylls template language). 

`Decision-{{Making}}` is rendered `Decision-`

We should remove the brackets then.
BetterBibLaTeX set them to ensure capitalization is preserved right when cited. 🤷‍♂️

**Review by ll-nick — APPROVED** [review]
Alright, I made the changes as discussed and removed the double curly braces.

I am not sure what the seconds change is about to be honest. At least to me all GitHub loges seem to be fine already. But I'll trust that you made that change for a reason so feel free to merge :+1:

**Pull request merged**